rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -1150,22 +1151,43 @@ def filterEnumerations(self, dataContainer, catName, atName, **kwargs):
 
     def __filterExperimentalResolution(self, dataContainer):
         """Collect resolution estimates from method specific sources."""
+
         rL = []
+        fL = []
+        eL = []
+
+        # _refine.pdbx_refine_id and refine.entry_id are joint composite keys for the refine category
+        # So, it is not possible to have multiple resolutions values for the same method in this category
+        # But _refine.pdbx_refine_id does not have a controlled vocabulary, and this can lead to accidental errors
         if dataContainer.exists("refine"):
             tObj = dataContainer.getObj("refine")
-            if tObj.hasAttribute("ls_d_res_high"):
+            if tObj.hasAttribute("ls_d_res_high") and tObj.hasAttribute("pdbx_refine_id"):
                 for ii in range(tObj.getRowCount()):
                     rv = tObj.getValue("ls_d_res_high", ii)
+                    rid = tObj.getValue("pdbx_refine_id", ii)
+                    rm = rid.upper()
```

Comment:
This is purely superficial, but using `rm` as a variable seems uncommon to me. Maybe `rM`? 

Also, just to confirm, `pdbx_refine_id` will always be populated with a value, right? (It won't ever be just a "." or "?") Although, I guess it wouldn't matter even if it did.

Reply:
Thanks @piehld. I'll change the variable name. 

Regarding `_refine.pdbx_refine_id`, this data item is part of the composite key for this category (along with refine.entry_id). So it will be populated as noted in my comment in that section of the code. The only problem here (discussed in detail in the JIRA ticket) is that this item does not have a controlled vocabulary that can lead to errors.  But this will hopefully be addressed in the near future.

Reply:
Yes, I tested this and it works.

Reply:
I see, thanks for clarifying. This sounds like the best possible strategy for handling this now then.